bukkit/src/main/java/com/leonardobishop/quests/bukkit/hook/versionspecific/VersionSpecificHandler14.java

```diff
@@ -0,0 +1,16 @@
+package com.leonardobishop.quests.bukkit.hook.versionspecific;
+
+import org.bukkit.event.inventory.InventoryType;
+
+public class VersionSpecificHandler14 extends VersionSpecificHandler9 implements VersionSpecificHandler {
+
+    @Override
+    public int getMinecraftVersion() {
+        return 14;
+    }
+
+    @Override
+    public boolean isFurnaceInventoryType(InventoryType type) {
+        return type == InventoryType.BLAST_FURNACE || type == InventoryType.FURNACE || type == InventoryType.SMOKER;
```

Comment:
New furnace types were added in 1.14
https://minecraft.fandom.com/wiki/Java_Edition_1.14#Blocks